sdk/avs/Microsoft.Azure.Management.Avs/src/Generated/Models/Cluster.cs

```diff
@@ -38,18 +38,18 @@ public Cluster()
         /// <param name="id">Resource ID.</param>
         /// <param name="name">Resource name.</param>
         /// <param name="type">Resource type.</param>
-        /// <param name="clusterSize">The cluster size</param>
         /// <param name="provisioningState">The state of the cluster
         /// provisioning. Possible values include: 'Succeeded', 'Failed',
         /// 'Cancelled', 'Deleting', 'Updating'</param>
+        /// <param name="clusterSize">The cluster size</param>
         /// <param name="clusterId">The identity</param>
         /// <param name="hosts">The hosts</param>
-        public Cluster(Sku sku, string id = default(string), string name = default(string), string type = default(string), int? clusterSize = default(int?), string provisioningState = default(string), int? clusterId = default(int?), IList<string> hosts = default(IList<string>))
+        public Cluster(Sku sku, string id = default(string), string name = default(string), string type = default(string), string provisioningState = default(string), int? clusterSize = default(int?), int? clusterId = default(int?), IList<string> hosts = default(IList<string>))
```

Comment:
This is Braking change: Swapping `clusterSize` and `provisioningState` places.

Please reorder this parameters in the swagger file.